### PR TITLE
Fix/B70

### PR DESCRIPTION
**Pull request by LeonMarcel-HD**

### CODE REVIEW

**Review by Gamer92000 — CHANGES_REQUESTED** [review]
don't do that please

File: Theme/package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
         "source": "Gamer92000.css",
         "apiVersion": 1,
         "image": "default1.png"
-      },
```
Comment (Gamer92000) [inline Theme/package.json]: Please remove this useless whitespace

File: Theme/Gamer92000.css [hunk 2]
```diff
@@ -470,7 +470,7 @@
     --talking-color: var(--outline-color);
 }
 
-:is(.tsv-icon-client-normal, .tsv-icon-client-voiceless, .tsv-icon-client-talking, .tsv-icon-client-detailed-normal, .tsv-icon-client-detailed-voiceless, .tsv-icon-client-detailed-talking) {
+:is(.tsv-icon-client-normal, .tsv-icon-client-voiceless, .tsv-icon-client-talking) {
```
Comment (Gamer92000) [inline Theme/Gamer92000.css:473]: What is this?

**Review by Gamer92000 — CHANGES_REQUESTED** [review]

File: Theme/Gamer92000.css (hunk 2 above)
Comment (LeonMarcel-HD) [reply Theme/Gamer92000.css:473]: Its from the past

**Review by Gamer92000 — APPROVED** [review]

**Pull request merged**